### ISSUE TITLE
Start-AutRun fails all tests all the time on Azure DevOps

### ISSUE THREAD

**Issue by Fleid** [Fleid/asa.unittest#9]
`Start-AutRun` runs fine locally but fails all tests on Azure DevOps (PS 5.1 and PS 7 / pwsh).

```PowerShell
...
VERBOSE: Ending Test Run with X errors
VERBOSE: 001
VERBOSE:    USAGE: jsondiffpatch left.json right.json
...
```

**Comment by Fleid** [comment]
Working on it, will be fixed in 1.0.5

In the meantime switch to PS7 when possible (`pwsh` at the top of build scripts).

**Comment by Fleid** [comment]
Can only reproduce in `windows-latest` agent of Azure DevOps, not locally.
Added verbose output in 1.0.5 to help investigate

**Comment by Fleid** [comment]
Solved with release 1.0.8
